hydrabus_framework/core/logger.py

```diff
@@ -0,0 +1,44 @@
+from tabulate import tabulate
```

Comment:
The logger should be places in the utils namespace. Indeed, it can be imported from modules right?

Reply:
Done [#8df850f](https://github.com/hydrabus-framework/framework/commit/8df850f2a9ecd72323d208ef8be2b72a621336ae)